src/lib/serviceRoutines/getContextEntityTypeAttribute.cpp

```diff
@@ -48,17 +56,29 @@ std::string getContextEntityTypeAttribute
   ParseData*                 parseDataP
 )
 {
-  std::string                          entityType     = (compV[0] == "v1")? compV[3] : compV[2];
-  std::string                          attributeName  = (compV[0] == "v1")? compV[5] : compV[4];
-  DiscoverContextAvailabilityRequest*  requestP       = &parseDataP->dcar.res;
-  EntityId                             entityId(".*", entityType, "true");
+  std::string  entityType     = (compV[0] == "v1")? compV[3] : compV[2];
+  std::string  attributeName  = (compV[0] == "v1")? compV[5] : compV[4];
+  std::string  answer;
 
   LM_T(LmtConvenience,
```

Comment:
Ok, let's do that. We can add them again if we need.

However, what about passed PRs? Are the LM_T removed them?